### PR TITLE
B2CA-1648: Port Flex

### PR DESCRIPTION
**Pull request by cedelavergne-ledger**
Port App to Flex.

This includes:
- Port NBGLv2
- Adapt tests using `scenario_navigator`
- Update version

# Checklist

- [X] App update process has been followed 
- [X] Target branch is `develop` 
- [X] Application version has been bumped

### CODE REVIEW

File: src/ui/address/address_ui_nbgl.c [hunk 1]
```diff
@@ -42,6 +42,11 @@ void display_address_confirmation_ui(char* address, action_t on_approve, action_
     rejection_action = on_reject;
     memset(full_address, 0, sizeof(full_address));
     strncpy(full_address, address, sizeof(full_address));
-    nbgl_useCaseAddressConfirmation((char*) full_address, confirmationChoiceClbk);
+    nbgl_useCaseAddressReview((char*) full_address,
+                              NULL,
+                              &C_icon_XRP_64px,
+                              "Verify Address",
```
Comment (tdejoigny-ledger) [inline src/ui/address/address_ui_nbgl.c]: "Verify XRP address" instead of "Verify Address"

**Review by tdejoigny-ledger — COMMENTED** [review]
a small change is required in codeql CI (add Flex target)

this line:
`sdk: ["$NANOS_SDK", "$NANOX_SDK", "$NANOSP_SDK", "$STAX_SDK"]`

**Comment by tdejoigny-ledger** [comment]
and same as for Tron it seems that the swap tests are not running for Flex target.
cc: @fbeutin-ledger

File: src/ui/address/address_ui_nbgl.c (hunk 1 above)
Comment (cedelavergne-ledger) [reply src/ui/address/address_ui_nbgl.c]: Done

**Review by tdejoigny-ledger — APPROVED** [review]

**Pull request merged**